Ardupilog.m

```diff
@@ -132,6 +133,9 @@ function delete(obj)
                     continue;
                 end
 
+                msgLen = obj.fmt_cell{i,3};
+                data = obj.isolateMsgData(msgId,msgLen,allHeaderCandidates);
+
```

Comment:
Any reason this went before the filter?
If it goes before, then it will be run regardless of whether this message should be filtered out or not.